dynamic-layers/openembedded-layer/recipes-extended/images/tb-minimal-efi-image.bb

```diff
@@ -13,6 +13,12 @@ IMAGE_INSTALL_append = " \
   grub-efi \
   dhcp-client \
   ipxe \
+  cbmem \
```

Comment:
The image recipes are already quite messy. Maybe that would be a proper time to use packagegroups and split the packages we need into a few named groups. Then, reuse those packagegroups in multiple images, rather than copying over a set of packages each time.

Reply:
Done.